game/game/api/shortcuts.py

```diff
@@ -0,0 +1,38 @@
+import logging
+
+import game.constants
+import game.packets
+from common.api_handlers import l2_request_handler
+from common.ctype import ctype
+from common.template import Parameter, Template
+from game.models import Character
+from game.models.structures.shortcut import Shortcut
+from game.request import GameRequest
+
+LOG = logging.getLogger(f"l2py.{__name__}")
+
+
+@l2_request_handler(
+    game.constants.GAME_REQUEST_SHORTCUT_REG,
+    Template(
+        [
+            Parameter(id="type", start=0, length=4, type=ctype.int32),
+            Parameter(id="slot", start="$type.stop", length=4, type=ctype.int32),
+            Parameter(id="id", start="$slot.stop", length=4, type=ctype.int32),
+        ]
+    ),
+    states="*",
+)
+async def request_short_cut_reg(request: GameRequest):
+    LOG.debug("saving shortcut", request)
+    character: Character = request.session.character
+    shortcut = Shortcut(
+        slot=ctype.int32(request.validated_data["slot"].value % 12),
+        page=ctype.int32(request.validated_data["slot"].value / 12),
+        type=ctype.int32(request.validated_data["type"]),
+        id=ctype.int32(request.validated_data["id"]),
```

Comment:
they are already `ctype.int32` not need to wrap once again
```suggestion
        type=request.validated_data["type"],
        id=request.validated_data["id"],
```

Reply:
ok, but how to be sure?
That is what I get, seems as int 4
![image](https://user-images.githubusercontent.com/10486621/188208964-94dd784e-cb20-40b1-a6e9-dc8e3ae07367.png)

Reply:
int: 4 == int 4 * 8 == int32

Reply:
4 here stands for bytes